api/src/wfl/sink.clj

```diff
@@ -280,17 +279,19 @@
   (jdbc/with-db-transaction [tx (postgres/wfl-db-config)]
     (jdbc/update! tx details {:consumed (utc-now)} ["id = ?" id])))
 
+(def ^:private active-job-query
+  (format
+   "SELECT id, job, workflow FROM %%s WHERE status IN %s ORDER BY id ASC"
```

Comment:
Just double-checking on the "%%s", is that intended?

Reply:
Yes - that's to escape the table name in the first format call. I can do `str/replace` if you prefer?

Reply:
No need, all good!